src/clojurians_log/db/queries.clj

```diff
@@ -102,6 +102,11 @@
        db
        name))
 
+(defn get-channel-id-by-name [channel-name]
+  ;; TODO: Need to read about any other optimized way like invert or somthing else
+  (key (first 
+        (filter #(= channel-name (val %)) (:chan-id->name @!indexes)))))
```

Comment:
We already have a mapping from name to id in index. This can be replaced by:

```
(get (:chan-name->id @!indexes) channel-name)
```

Reply:
damn, i searched for that and overlooked this one. should be more easy with that now.